### PR TITLE
Add program and participants

### PR DESCRIPTION
**Pull request by thofma**
@fingolfin @fieker @ThomasBreuer

### CODE REVIEW

File: meetings/2021-09/program.md [hunk 1]
```diff
@@ -9,10 +9,17 @@ meeting_nr: 202109
...
+about computational algebra, both theoretical and practical. The schedule
+will encompass:
+- introductory courses by experts on select topics of computational algebra by
+  - [Eamonn O'Brien](https://www.math.auckland.ac.nz/~obrien/) on algorithmic group theory
+  - [Clément Pernet](https://www-ljk.imag.fr/membres/Clement.Pernet/index.html) on computational linear algebra
 - introductory sessions on working with git, Julia, OSCAR
 - hand-on training sessions implementing selected algorithms in Julia and OSCAR
 
-Details will be announced here and via the SFB mailing list at a later point.
+The introductory courses require only few perquisites and will be accompanied
```
Comment (ThomasBreuer) [inline meetings/2021-09/program.md]: prerequisites

File: meetings/2021-09/program.md [hunk 2]
```diff
@@ -9,10 +9,17 @@ meeting_nr: 202109
 * Finish: Friday, September 10
 
 The aim of the summer school is to teach participants knowledge
-about computational algebra, both theoretical and practical. To this end,
-we plan the following:
-- introductory courses by experts on select topics of computational algebra
+about computational algebra, both theoretical and practical. The schedule
+will encompass:
+- introductory courses by experts on select topics of computational algebra by
```
Comment (ThomasBreuer) [inline meetings/2021-09/program.md]: selected topics

**Review by ThomasBreuer — APPROVED** [review]
Thanks. There are just two typos (one already in the previous version), see the comments.

File: meetings/2021-09/program.md (hunk 1 above)
Comment (thofma) [reply meetings/2021-09/program.md]: Apparently, "perquisites" is also a word.

**Pull request merged**